pr_testing/test_multiple_prs.sh

```diff
@@ -1141,6 +1119,16 @@ GENERAL_ERRORS=`grep "ALL_OK" $WORKSPACE/build.log` || true
 
 rm -f $WORKSPACE/deprecated-warnings.log
 get_compilation_warnings $WORKSPACE/build.log > $WORKSPACE/all-warnings.log
+
+pushd ..
+scram b echo_SCRAM_TOOL_HOME
+test -e config/SCRAM/findDependencies.py
+SCRAM_TOOL_HOME=`scram b echo_SCRAM_TOOL_HOME 2>/dev/null | tail -1 | cut -d' ' -f3`
+mkdir -p etc/dependencies
+SCRAM_TOOL_HOME=$SCRAM_TOOL_HOME ./config/SCRAM/findDependencies.py -rel `pwd` -arch ${SCRAM_ARCH}
```

Comment:
@iarspider , this will not work for old perl based releases . See https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_11_0_X/master/scram-project-build.file#L210 . So you need to check if `config/scram_version` has scram version V3_* then use findDependencies.py otherwise use findDependencies.pl ( with correct parameters)

Reply:
Noted.

Reply:
@iarspider , we still need to fix this perl based scram